### PR TITLE
Newtx

### PR DESCRIPTION
**Pull request by c4t-ag**
# AddValidator / AddDelegator Changes
### Main changes: (commit 1)
- [TX]: remove unsupported AddValidatorTx properties
- [TX]: rename AddDelegatorTx -> AddDepositTx

### Cleanups: (commit 2)
- [CI] run on kopernikus pull and push
- [Examples]: fix baseTx-avax.ts sample implementation
- [BUILD]: remove typings folder from prettier

### CODE REVIEW

File: src/apis/platformvm/constants.ts [hunk 1]
```diff
@@ -58,4 +53,20 @@ export class PlatformVMConstants {
   static ASSETNAMELEN: number = 128
 
   static ADDRESSLENGTH: number = 20
+
+  // Get the lates possible version
```
Comment (Chriserus) [inline src/apis/platformvm/constants.ts]: Typo, `latest`*

File: tests/apis/platformvm/api.test.ts [hunk 2]
```diff
@@ -2381,9 +2387,9 @@ describe("PlatformVMAPI", (): void => {
       const txu1Outs: TransferableOutput[] = (
         txu1.getTransaction() as AddValidatorTx
       ).getOuts()
-      const txu1Stake: TransferableOutput[] = (
-        txu1.getTransaction() as AddValidatorTx
-      ).getStakeOuts()
+      //const txu1Stake: TransferableOutput[] = (
+      //  txu1.getTransaction() as AddValidatorTx
+      //).getStakeOuts()
```
Comment (Chriserus) [inline tests/apis/platformvm/api.test.ts:2380]: Do we need to leave that as commented in?

File: tests/apis/platformvm/api.test.ts [hunk 3]
```diff
@@ -2404,13 +2410,13 @@ describe("PlatformVMAPI", (): void => {
...
-      for (let i: number = 0; i < txu1Stake.length; i++) {
-        staketotal = staketotal.add(
-          (txu1Stake[i].getOutput() as AmountOutput).getAmount()
-        )
-      }
+      //for (let i: number = 0; i < txu1Stake.length; i++) {
+      //  staketotal = staketotal.add(
+      //    (txu1Stake[i].getOutput() as AmountOutput).getAmount()
+      //  )
+      //}
```
Comment (Chriserus) [inline tests/apis/platformvm/api.test.ts:2407]: Same for this comment. Is it necessary to leave it in?

File: tests/apis/platformvm/api.test.ts [hunk 4]
```diff
@@ -2422,8 +2428,8 @@ describe("PlatformVMAPI", (): void => {
 
       expect(intotal.toString(10)).toBe("4000000000")
       expect(outtotal.toString(10)).toBe("1000000000")
-      expect(staketotal.toString(10)).toBe("3000000000")
-      expect(totaltotal.toString(10)).toBe("4000000000")
+      //expect(staketotal.toString(10)).toBe("3000000000")
```
Comment (Chriserus) [inline tests/apis/platformvm/api.test.ts:2419]: Same with this comment

File: src/apis/platformvm/api.ts [hunk 5]
```diff
@@ -1577,7 +1577,7 @@ export class PlatformVMAPI extends JRPCAPI {
       )
     }
 
-    const builtUnsignedTx: UnsignedTx = utxoset.buildAddDelegatorTx(
+    const builtUnsignedTx: UnsignedTx = utxoset.buildAddDelegatorTxx(
```
Comment (Chriserus) [inline src/apis/platformvm/api.ts]: What is the intent of changing everything related to `DelegatorTx` to `DelegatorTxx`?

Comment (c4t-ag) [reply src/apis/platformvm/api.ts]: Delegating will be replaced by Deposit with a different logic, just wanted to leave it in to better find code places.
I'll rename it to Deposit, then it makes things clearer.

File: tests/apis/platformvm/api.test.ts (hunk 4 above)
Comment (c4t-ag) [reply tests/apis/platformvm/api.test.ts:2419]: Should remain to replace them with LockedIn / LockedOut logic as soon they are in.

File: tests/apis/platformvm/api.test.ts (hunk 3 above)
Comment (c4t-ag) [reply tests/apis/platformvm/api.test.ts:2407]: Should remain to replace them with LockedIn / LockedOut logic as soon they are in.

File: tests/apis/platformvm/api.test.ts (hunk 2 above)
Comment (c4t-ag) [reply tests/apis/platformvm/api.test.ts:2380]: Should remain to replace them with LockedIn / LockedOut logic as soon they are in.

**Review by Chriserus — APPROVED** [review]

**Pull request merged**